src/tests/test_command.py

```diff
@@ -78,7 +78,8 @@ def test_parsed_sql_when_using_with(ip, sql_magic):
         sql_magic, ip.user_ns, line="--with author_one", cell="SELECT * FROM author_one"
     )
 
-    sql = "WITH author_one AS (SELECT * FROM author LIMIT 1) SELECT * FROM author_one"
+    sql = "WITH `author_one` AS (\n\n        SELECT * FROM author LIMIT 1\n        )\n\
```

Comment:
Why did you change the original query? Can we parametrize it and test both?

Reply:
I think because it's we are using sqlite engine to test, and sqlite actually [support backtick symbol](https://github.com/tobymao/sqlglot/blob/main/sqlglot/dialects/sqlite.py#L27), so we need to change the test case here